tests/fps_tests.py

```diff
@@ -704,6 +706,184 @@ def test_expected_esld(self):
         self.assertEqual(loaded_sets, expected_sets)
         self.assertEqual(fp.error_list, [])
 
+class TestFindDiff(unittest.TestCase):
+    def test_unchanged_sets(self):
+        old_sets = {
+            'https://primary.com': 
+            FpsSet(
+                    primary="https://primary.com", 
+                    associated_sites=None,
+                    service_sites=None,
+                    ccTLDs={
+                        "https://primary.com": ["https://primary.ca"]
+                    }
+                    )
+        }
+        new_sets = {
+            'https://primary.com': 
+            FpsSet(
+                    primary="https://primary.com", 
+                    associated_sites=None,
+                    service_sites=None,
+                    ccTLDs={
+                        "https://primary.com": ["https://primary.ca"]
+                    }
+                    )
+        }
+        diff_sets, subtracted_sets = find_diff_sets(old_sets, new_sets)
+        self.assertEqual(diff_sets, {})
+        self.assertEqual(subtracted_sets, {})
+
+    def test_added_set(self):
+        old_sets = {
+            'https://primary.com': 
+            FpsSet(
```

Comment:
Since we're relying on `FpsSet.__eq__`, we probably also want to add test cases for that, separately, to ensure that it checks all of the fields, handles `FpsSet` instances that are equivalent but [not the same instance](https://docs.python.org/3/reference/expressions.html#is) correctly (i.e. `fps1 is fps2` is false but `fps1 == fps2` is true), etc.

Reply:
Done